safe-global/web-core
# fix: spending limit validation

## What it solves

Resolves #1677

## How this PR fixes it

The multisig option no longer validates according to a spending limit. When changing the transaction type, the amount is now again validated as well.

## How to test it

Create a transaction and select "Multisig Transaction" when a spending limit is set. Observe no validation error when entering an amount above the spending limit. Switching to a "Spending Limit Transaction" should validate and show an error.

## Screenshots

![spending limit](https://user-images.githubusercontent.com/20442784/219622579-e3788a26-3971-48a7-9db7-b01f0f0dca9b.gif)
